infra/app/env-template/outputs.tf

```
@@ -0,0 +1,4 @@
+output "public_endpoint" {
+  description = "The DNS name of the load balancer that is created in the service module."
```

tiny nit, but a service "endpoint" includes the protocol e.g. "http://helloworld.com" whereas the dns name is just "helloworld.com", so I think we should tweak the description just a tad
```suggestion
  description = "The public endpoint of the application service's load balancer"
```